MuxExoPlayer/build.gradle

```diff
@@ -123,8 +123,8 @@ dependencies {
     r2_12_1Api 'org.checkerframework:checker-qual:2.5.2'
     r2_12_1_adsApi 'com.google.android.exoplayer:exoplayer:2.12.1'
     r2_12_1_adsApi 'org.checkerframework:checker-qual:2.5.2'
-    compileOnly 'com.google.ads.interactivemedia.v3:interactivemedia:3.9.0'
-    compileOnly 'com.google.android.gms:play-services-ads:15.0.1'
+    compileOnly 'com.google.ads.interactivemedia.v3:interactivemedia:3.20.1'
```

Comment:
Or is it bumping the dependency in the compile-only flag?

Reply:
Bumping the dependency version is not mandatory, do we want to keep old versions ?

Reply:
If it doesn't hurt anything, I don't think it matters, honestly.